pyquil/wavefunction.py

```diff
@@ -145,28 +146,42 @@ def pretty_print_probabilities(self, decimal_digits=2):
                 outcome_dict[outcome] = prob
         return outcome_dict
 
-    def pretty_print(self, decimal_digits=2):
+    def pretty_print(self, decimal_digits=2, ancillas: List[int] = None):
         """
         Returns a string repr of the wavefunction, ignoring all outcomes with approximately zero
         amplitude (up to a certain number of decimal digits) and rounding the amplitudes to
         decimal_digits.
 
         :param int decimal_digits: The number of digits to truncate to.
+        :param List[int] ancillas: The list of indices for ancillary qubits to omit.
         :return: A dict with outcomes as keys and complex amplitudes as values.
         :rtype: str
         """
+
+        if ancillas is None:
+            ancillas = []
+
+        mask = 0
+        for ancilla in ancillas:
+            mask += 2**ancilla
```

Comment:
```suggestion
            mask |= (1 << ancilla)
```

`style_points += 1`

Reply:
otoh maybe a mathematician will say `style_points -= 1`, but don't listen to them, follow ur ❤️ .